addons/block_code/blocks/communication/area2d_on_entered.tres

```diff
@@ -10,8 +10,8 @@ description = ""
 category = "Communication | Methods"
 type = 1
 variant_type = 0
-display_template = "On [body: OBJECT] entered"
-code_template = "func _on_body_entered(body: Node2D):
+display_template = "When entering collision with [something: OBJECT]"
```

Comment:
I don't like the "entering collision with" / "leaving collision with" pair. I don't think collisions are something you leave. It's awkward to phrase this because we don't have a [subject][]

[subject]: https://en.wikipedia.org/wiki/Subject_(grammar)

Brainstorming:

- When colliding with [something] / When no longer colliding with [something]
- When this node collides with [something] / When this node stops colliding with [something]
- When collision with [something] starts / When collision with [something] ends
- When [something] collides with this node / When [something] stops colliding with this node

Not sure I like these much better…

Reply:
I like the second one of those; "When this node collides with / stops colliding with [something]". Alternatively, we could write these in the first person, like "When I collide with", which I think is the nicest to read here but stylistically it's tricky because we don't want to be writing _everything_ in first person.

Reply:
This is a good point, should we switch everything to first person? Ie. "Remove me" instead of "Remove"? I'm open to do it.